### DISCUSSION TITLE
Error type Module '"@ioc:Adonis/Addons/Mail"' has no exported member 'MailConfig'

### DISCUSSION THREAD

**Discussion by brunodsazevedo** [adonisjs/core#3714] [Help] [answered]
I installed and configured Mailer, according to the documentation, however, in my VSCode it shows this typing error

![image](https://user-images.githubusercontent.com/58368716/166919030-bb99c180-4320-473a-aff4-770f64e7b356.png)

![image](https://user-images.githubusercontent.com/58368716/166919243-f506818b-2c39-492e-bccc-bc14eeb8ca7f.png)

How can I solve this typing problem?

**Comment by Julien-R44** [comment, marked as answer]
It's probably your editor that didn't pick up the new types. So just restart your editor and it should be good
